array/zfp/pointer1.h

```diff
@@ -2,6 +2,7 @@
 class pointer {
 public:
   pointer() : ref(0, 0) {}
+  pointer(std::nullptr_t n) : ref(0, 0) {}
```

Comment:
std::nullptr_t is C++11 while zfp is C++98 compliant.  If this is absolutely necessary for RAJA, then appropriate ifdefs are needed to hide this ctor.

Reply:
It is used call `std::vector<ptr_t>::resize` in the same way whether `ptr_t = double *` or `ptr_t = zfp::array::pointer`.

I will add the proper `#ifdef`

Reply:
zfp's pointers are null initialized by default, so `pointer()` and `pointer(nullptr)` will achieve the same.  Is this ctor a (Kripke/Raja) requirement for a template interface?

If needed, I would still leave the unused parameter name off to avoid potential compiler warnings.

Reply:
It is needed to match the initialization of a `double *` to nullptr (minimizes template specialization). I removed the name.